plugins/usbpro/OpenDeckWidget.cpp

```diff
@@ -0,0 +1,84 @@
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * OpenDeckWidget.cpp
+ * The OpenDeck Widget.
+ * Copyright (C) 2022 Igor Petrovic
+ */
+
+#include <vector>
+
+#include "ola/Constants.h"
+#include "ola/Logging.h"
+#include "ola/util/Utils.h"
+#include "plugins/usbpro/OpenDeckWidget.h"
+
+namespace ola {
+namespace plugin {
+namespace usbpro {
+
+using ola::utils::SplitUInt16;
+
+/**
+ * OpenDeckWidget Constructor
+ */
+OpenDeckWidget::OpenDeckWidget(
+  ola::io::ConnectedDescriptor *descriptor)
+    : GenericUsbProWidget(descriptor) {
+}
+
+bool OpenDeckWidget::IsBufferDiff(const DmxBuffer &buffer) {
+  return buffer != internal_buffer;
+}
```

Comment:
Great minds! I literally just came here to write this after reviewing #1766 who's doing a very similar thing...